package.json

```diff
@@ -18,6 +18,16 @@
   },
   "author": "",
   "license": "MIT",
+  "files": [
+    "LICENSE",
+    "README.md",
+    "lib/",
+    "examples/"
```

Comment:
Perhaps we *don't* include examples?  

...e.g.: to prevent a consumer from importing / running express?

Reply:
👍 I think anyone wanting help will visit our github repo.